Classify Satellites #39

zapper-95 opened the pull request:
I added the satellites Phobos, Tethys, and Ganymede and trained the model on them.
I managed to get fairly good confidence values for the model on each of them for the examples shown and an overall test accuracy of 85.7%

![tethys](https://user-images.githubusercontent.com/92369126/166110175-410d97c4-7af0-4c80-aba5-4afd9255ee80.png)
![ganymede](https://user-images.githubusercontent.com/92369126/166110223-a44500c1-6352-4010-b708-756107cf175f.png)
![phobos](https://user-images.githubusercontent.com/92369126/166110248-6daf4068-23a0-4224-a715-e5280a96c21a.png)
![85 7](https://user-images.githubusercontent.com/92369126/166110263-7991e71d-0b96-453a-82d4-3c8597f1eabc.png)

zapper-95 committed:
Added more satellites and tried to improve the mixture of images in the training to testing set

zapper-95 committed:
removed duplicate uranus files

ritwik12 reviewed (changes requested):
@zapper-95 Thanks for creating this PR. There are few issues with the PR. Can you please remove ".vs" directory and addition to "/bottlenecks" in your PR?

Also, you will have to add info to hub/examples/image_retraining/display_info.yml regarding all new additions. And if you can make changes to the readme also about the same. Please do that too.

zapper-95 committed:
Removed .vs and bottlenecks and added more information to README and retrained_labels

zapper-95 commented:
@ritwik12 I applied the changes you said about: deleted the .vs and bottleneck folder and added info to display_info and the README in my forked version. I hope this is okay, but if you think there is any more issues please let me know.

ritwik12 commented:
@zapper-95 Thanks for that.

There is still problem, I guess while deleting those you deleted a lot of project files too. it is showing 1974 files changed for this PR. please rebase this and add only your changes.
